assets/app/scripts/controllers/settings.js

```diff
@@ -26,7 +26,31 @@ angular.module('openshiftConsole')
     ProjectsService
       .get($routeParams.project)
       .then(_.spread(function(project, context) {
-        $scope.project = project;
+        angular.extend($scope, {
+          project: project,
+          show: { display: true, form: false, actions: true },
+          toggle: function() {
+            $scope.show.display = !$scope.show.display;
+            $scope.show.form = !$scope.show.form;
+          },
+          update: function() {
+            $scope.toggle();
+            $scope.show.actions = false;
+            ProjectsService
+              .update($routeParams.project, angular.copy(project))
```

Comment:
We'll also show the updated values even when the update fails as written, correct?